java/src/main/java/com/powsybl/python/network/NetworkCFunctions.java

```diff
@@ -870,4 +870,15 @@ public static DataframeMetadataPointer getLineFeederBaysMetadata(IsolateThread t
         });
     }
 
+    @CEntryPoint(name = "getCouplingDeviceCreationMetadata")
+    public static DataframeMetadataPointer getCouplingDeviceCreationMetadata(IsolateThread thread, ExceptionHandlerPointer exceptionHandlerPtr) {
```

Comment:
as in the other pr it would be great to refactor this method with other metadata method